<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(206, 238, 255);
font: 700 10pt &quot;Comic Sans MS&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-8</x>
      <y>35</y>
      <width>541</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 244, 84);
font: 700 bold 13pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string>MÃ HÓA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>85</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>BẢN RÕ</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="message1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>331</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(246, 255, 255);
border-radius: 10px;
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>KHÓA K</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="key1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(246, 255, 255);
border-radius: 10px;
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="file1">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>140</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(148, 106, 255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>CHỌN FILE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="encript">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>300</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(148, 106, 255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>MÃ HÓA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>BẢN MÃ</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="cipher1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>331</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(246, 255, 255);
border-radius: 10px;
</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="file2">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>140</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(148, 106, 255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>CHỌN FILE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>539</x>
      <y>35</y>
      <width>531</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 244, 84);
font: 700 bold 13pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string>GIẢI MÃ</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="key2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>250</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(246, 255, 255);
border-radius: 10px;
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>230</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>KHÓA K</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="cipher2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>110</y>
      <width>331</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(246, 255, 255);
border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>90</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>BẢN MÃ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decript">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>300</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(148, 106, 255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>GIẢI MÃ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>350</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>BẢN RÕ</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="message2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>370</y>
      <width>331</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(246, 255, 255);
border-radius: 10px;
</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="reset">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>510</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 0);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>RESET</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>510</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 0);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>THOÁT</string>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>528</x>
      <y>30</y>
      <width>16</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 173, 8);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="Save">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>400</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(148, 106, 255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>SAVE FILE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Save_3">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>380</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(148, 106, 255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>SAVE FILE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="kngaunhien">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>250</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(148, 106, 255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>NGẪU NHIÊN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="kngaunhien_2">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>250</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(148, 106, 255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>NGẪU NHIÊN</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-6</y>
      <width>1081</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 255);
font: 700 bold 19pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string>THUẬT TOÁN DES</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1065</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
